kwords.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+
+url = "http://0.0.0.0:5000/"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The URL "http://0.0.0.0:5000/" is hardcoded which might cause issues if the server's address changes in the future. It would be better to store this as an environment variable or a configuration parameter.
```diff
- url = "http://0.0.0.0:5000/"
+ import os
+ url = os.getenv('SERVER_URL', 'http://0.0.0.0:5000/')
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->